pom.xml

```diff
@@ -50,7 +50,7 @@
         <maven.sonar.plugin.version>3.3.0.603</maven.sonar.plugin.version>
         <maven.jacoco.plugin.version>0.8.5</maven.jacoco.plugin.version>
         <jsr107.api.version>1.1.1</jsr107.api.version>
-        <bytebuddy.version>1.10.9</bytebuddy.version>
+        <bytebuddy.version>1.10.10</bytebuddy.version>
```

Comment:
@leszko 
Is bytebuddy used by hazelcast-aws?

Reply:
Actually not. Let me close this this PR. And I created this one: #173